### ISSUE TITLE
JSON.OBJKEYS/JSON.OBJLEN should return an empty array when being used with "multi path"

### ISSUE THREAD

**Issue by leibale** [RedisJSON/RedisJSON#509]
```
> JSON.SET foo $ {}
OK

> JSON.OBJKEYS foo $
1) (empty array)
> JSON.OBJKEYS bar $
(nil)
> JSON.OBJKEYS bar
(nil)

> JSON.OBJKEYS foo $
1) (empty array)
> JSON.OBJKEYS bar $
(nil)
> JSON.OBJKEYS bar
(nil)
```

**Comment by oshadmi** [comment]
@leibale Will be fixed by #548 by throwing an exception on missing key or path

**Comment by chayim** [comment]
I'd be in favour of our always returning an array - and sometimes that array having a length of one. At least it would be consistent from the user's perspective.  Unpopular - yes. Thoughts?

**Comment by oshadmi** [comment]
Updating according to #553

- Throw an exception on non-existing key

- Return empty array on non-existing path

**Comment by oshadmi** [comment]
Fixed by #553
